### ISSUE TITLE
Ethereum Wallet doesn't sync

### ISSUE THREAD

**Issue by tskol** [ethereum/mist#3836]
Hello!

I see my transaction on https://etherscan.io but can't recieve in my Ethereum Wallet.
I see always "Waiting for blocks"
I tried unsusseccfully sync for more than week.

Sometime I tried to sync with Light client (beta). Same result.

Screenshots: https://prnt.sc/j4kgw7
https://prnt.sc/j4kh5f

Could someone help me please.

* [Y] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.10.0`
OS & Version: windows 7
Node version: `geth 1.8.2`
Number of blocks synchronized: now close to 100K items needed to sync.
```

**Comment by philsmd** [comment]
It should work pretty fast in light mode, but there are currently not too many nodes that serve light clients (but it's increasing).
I think the problem is that you see 0 peers. Did you make sure that the firewall settings are okay and that you have internet access etc? Did you try to restart mist/ethereum wallet?
You could also try to use geth directly (it should be in your %APPDATA%\Ethereum Wallet\binaries\Geth\unpacked\ folder, you need to run it in cmd, something like "geth --syncmode light console", without quotes)

**Comment by AmxAmx** [comment]
Last Week Light Client dosen Sync at all!!!! Now go download full node....

**Comment by AmxAmx** [comment]
Really bored of this

**Comment by xixou2** [comment]
+1

**Comment by tskol** [comment]
I still can't download and count is growing. Generally, I see the screen attached
![e](https://user-images.githubusercontent.com/38337553/39111254-b25d6646-46fe-11e8-92a8-6a79f7dd0011.png)

**Comment by AmxAmx** [comment]
I open C:\Users\***\AppData\Roaming\Ethereum Wallet\binaries\Geth\unpacked\geth.exe and wait 5 days for download!! After that i open ethereum wallet.exe and use it.

**Comment by tskol** [comment]
Yes, finally I downloaded all. And sync now works regullary too.
Thanks everyone for help!!!
I'm almost happy but my transaction is not in my wallet.  Whole sync done, but I still see 0 in my account and no my transaction...
https://etherscan.io show the transaction sucessfully completed .
I can't uderstand this, How can it be possible?

**Comment by milyInzhoop** [comment]
same issues...
Now, for me, Mist is Synced but is impossible to send transactions, they will be not performed really

**Comment by lorezyra** [comment]
Related: https://github.com/ethereum/mist/issues/2466

Please define OS and version.

**Comment by tskol** [comment]
Version: `0.10.0`
OS & Version: windows 7
Node version: `geth 1.8.2`

**Comment by tskol** [comment]
Unbelievable!
Month passed through, stupid wallet still didn't download the transaction!
System doesn't work, it's just fiction adn fraud.

Is there another way to receive coins of existed ether account ?

![wal](https://user-images.githubusercontent.com/38337553/40166640-3b98d196-59e9-11e8-8167-3426c2ebef3e.gif)

**Comment by ryanio** [comment]
Hey guys sorry about the issues, we are working on a better node management system in development right now so you can use a remote node on start, then switch to local when sync'd up.

Until the new version is ready, you can use [MetaMask](https://metamask.io/) and [MyEtherWallet.com](https://myetherwallet.com/) or [MyCrypto.com](https://mycrypto.com/) to access your accounts and funds. If you just want to view your up-to-date balances, you can check using [Etherscan](https://etherscan.io).

**Comment by ITBOOBOO** [comment]
Funny

After a year or so of dealing as a user with this crud code I could have a tonne of profanities instead of funny but...

Funny will do for now.

**Comment by tskol** [comment]
Just thinking... Official wallet, main etherium tool doesn't work for a lot of time. 
If so what's etherium mean itself? how ether can be traded in markets with this position?
Are all marketing words about ether, smart contracts and so on just dust ?

**Comment by skoupro** [comment]
Ethereum wallet is crap. It's sad really that they can't (or won't) make a lightweight wallet. I've seen 15 year old's make great lightweight wallets for their shit-coins that makes me wonder.

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
